api/v1beta1/machine_types.go

```diff
@@ -230,10 +230,22 @@ type MachineStatus struct {
 	// Conditions defines current service state of the Machine.
 	// +optional
 	Conditions Conditions `json:"conditions,omitempty"`
+
+	// Deletion is the deletion state of the Machine.
```

Comment:
Where did we land with getting the godocs to start with the serialised versions?

Perhaps a little more context might be useful?

```suggestion
	// deletion contains information relating to removal of the Machine.
	// Only present when the Machine has a deletionTimestamp and is being removed from the cluster.
```

Reply:
> Where did we land with getting the godocs to start with the serialised versions?

First time I'm hearing about this :). godoc conventions suggest it should be upper case, right?

Reply:
I can't see it explicitly called out in the API conventions from upstream, but, take a look at the godoc across the core types (and I know the upstream API reviewers enforce this from experience), https://github.com/kubernetes/api/blob/master/core/v1/types.go, all of the godocs on fields start with the json tag version of the field name, so that when docs are generated, they represent the serialised versions of the strings

Reply:
Makes sense. Thank you!

@chrischdi Can you please change it here and also open an help-wanted issue so we change it everywhere ?

Reply:
xref: 
https://github.com/kubernetes-sigs/cluster-api/issues/11238